Dockerfile

```diff
@@ -2,15 +2,16 @@ FROM node:16-alpine AS builder
 WORKDIR /app
 COPY . /app
 
-RUN apk add --no-cache libc6-compat yarn
-# RUN yarn --frozen-lockfile
-RUN yarn
+RUN apk add --no-cache libc6-compat
+
+RUN npm ci
```

Comment:
npm ci is equivalent to yarn --frozen-lock in that it installs what it finds in the `package-lock.json`, not the `package.json` file, hence version locking package installs